src/main/resources/client_secrets.json

```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"298755462-7fnqhho2db1b0mb5i6o7irgg8v7om76g.apps.googleusercontent.com","project_id":"step170-2020","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"uO1lK-fFXtfRCtpbsBoRIGgV","redirect_uris":["https://step170-2020.uc.r.appspot.com","https://8080-326c8364-c923-46b5-ada4-ce21a494db67.us-west1.cloudshell.dev","https://8080-06212729-9771-4457-b979-a463e6c0b02f.us-east1.cloudshell.dev"],"javascript_origins":["https://step170-2020.uc.r.appspot.com","https://8080-326c8364-c923-46b5-ada4-ce21a494db67.us-west1.cloudshell.dev","https://8080-06212729-9771-4457-b979-a463e6c0b02f.us-east1.cloudshell.dev"]}}
```

Comment:
This file shouldn't be pushed to github; it has our client id. Actually, just add the entire resource folder to your .gitignore and remove unstage the folder from git.

Reply:
Opps. Sorry. Made a type in gitignore to ignore this file.

Reply:
Gotcha!

Reply:
This file is still showing.

Reply:
After adding resource folder to gitignore, you also need to remove this file from the commit, otherwise it will still be pushed.